### PR TITLE
Latest gcc Travis checks

### PR DESCRIPTION
**Pull request by CodeMaxx**
Travis does an additional compiling with gcc 5.2 and g++ 5.2(latest versions)
#684

### CODE REVIEW

File: bin/install_travis.sh [hunk 1]
```diff
@@ -90,5 +93,5 @@ if [[ "${WITH_PIRANHA}" == "yes" ]]; then
     git clone https://github.com/bluescarni/piranha;
     cd piranha && mkdir build && cd build;
     cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$our_install_dir -DBUILD_TESTS=no ../ && make -j8 install && cd ../..;
-fi
+fi
```
Comment (certik) [inline bin/install_travis.sh]: Any reason to add this whitespace?

**Comment by certik** [comment]
It seems to be working. +1 to merge, except the minor comment above.

File: bin/install_travis.sh (hunk 1 above)
Comment (CodeMaxx) [inline bin/install_travis.sh]: No the whitespace is not required.

File: bin/install_travis.sh [hunk 2]
```diff
@@ -23,6 +23,9 @@ if [[ "${TRAVIS_OS_NAME}" == "linux" ]] && [[ "${CC}" == "gcc" ]]; then
     if [[ "${WITH_PIRANHA}" == "yes" ]]; then
         export CC=gcc-4.8
         export CXX=g++-4.8
+    elif [[ "${WITH_LATEST_GCC}" == "yes" ]]; then
+    	export CC=gcc-5
```
Comment (isuruf) [inline bin/install_travis.sh]: Use spaces here instead of tabs to match the indentation of line 25

**Comment by isuruf** [comment]
This seems to be using gcc-5.2, so can you change the commit message to reflect that?

**Comment by CodeMaxx** [comment]
It downloaded as version 5.3 on my machine. Where does it say 5.2?

File: bin/install_travis.sh (hunk 1 above)
Comment (CodeMaxx) [inline bin/install_travis.sh]: In #678  I ignored `.sh` files of the `bin` folder . Should I remove that exception and check for `.sh` files too? That would have prevented this error.

**Comment by isuruf** [comment]
That's because you are using Ubuntu trusty. For precise, it's gcc-5.2. https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test?field.series_filter=precise

**Comment by CodeMaxx** [comment]
@isuruf  Got it ! Thanx for notifying.

**Comment by CodeMaxx** [comment]
@isuruf @certik  
Ping

**Comment by isuruf** [comment]
@CodeMaxx, thanks for the PR

**Comment by CodeMaxx** [comment]
@isuruf  Thanx for merging.

**Pull request merged**